fe/src/main/java/org/apache/doris/analysis/CreateTableStmt.java

```diff
@@ -412,9 +412,13 @@ public void analyze(Analyzer analyzer) throws UserException {
 
                             // key columns and none/replace aggregate non-key columns support
                             if (indexDef.getIndexType() == IndexDef.IndexType.BITMAP) {
-                                    if (!(colType == PrimitiveType.TINYINT || colType == PrimitiveType.SMALLINT
-                                                  || colType == PrimitiveType.INT || colType == PrimitiveType.BIGINT ||
-                                                  colType == PrimitiveType.CHAR || colType == PrimitiveType.VARCHAR)) {
+                                    if (!(colType == PrimitiveType.TINYINT || colType == PrimitiveType.SMALLINT ||
```

Comment:
use isNumericType(), not list all type

Reply:
it dose not support double and float and double， so cannot use  isNumericType()